### ISSUE TITLE
Proposal for asyncio: SubprocessTransport.detach() to detach a process from a transport

### ISSUE THREAD

**Issue by Martiusweb** [python/cpython#67728]
BPO | [23540](https://bugs.python.org/issue23540)
--- | :---
Nosy | @gvanrossum, @vstinner, @1st1, @Martiusweb
Files | <li>[add-detach-to-subprocess_transport.patch](https://bugs.python.org/file38263/add-detach-to-subprocess_transport.patch "Uploaded as text/plain at 2015-02-27.18:11:08 by @Martiusweb"): First try: add a detach() method to SubprocessTransport.</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2015-02-27.18:11:10.238>
labels = ['type-feature', 'expert-asyncio']
title = 'Proposal for asyncio: SubprocessTransport.detach() to detach a process from a transport'
updated_at = <Date 2015-02-27.18:11:10.238>
user = 'https://github.com/Martiusweb'
```

bugs.python.org fields:
```python
activity = <Date 2015-02-27.18:11:10.238>
actor = 'martius'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['asyncio']
creation = <Date 2015-02-27.18:11:10.238>
creator = 'martius'
dependencies = []
files = ['38263']
hgrepos = []
issue_num = 23540
keywords = ['patch']
message_count = 1.0
messages = ['236808']
nosy_count = 4.0
nosy_names = ['gvanrossum', 'vstinner', 'yselivanov', 'martius']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue23540'
versions = ['Python 3.3', 'Python 3.4', 'Python 3.5']
```

</p></details>

**Comment by kumaraditya303** [comment]
No activity for ~7 years, I am unclear about the use case for this, marking pending.

**Comment by kumaraditya303** [comment]
I am closing this as the use cases requires fork which asyncio doesn't properly supports and there are no comments after the initial issue.
